src/routes/open/components/SafeOwnersConfirmationsForm/index.tsx

```diff
@@ -203,7 +203,7 @@ const SafeOwnersForm = (props): React.ReactElement => {
                   testId={`create-safe-owner-name-field-${index}`}
                 />
               </Col>
-              <Col className={classes.ownerAddress} xs={7}>
+              <Col className={classes.ownerAddress} spellcheck="false" xs={7}>
```

Comment:
Col is a `div` element, it's not an input

You should put it to the generic AddressInput component

Reply:
Totally, you're so right! Thank you @mikheevm